src/ui/CardContent/CardContent.css

```
@@ -0,0 +1,45 @@
+.CardContent {
+  height: 100%;
+  padding: 1.5rem 2rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 1rem;
+  transition: background-color 0.3s ease-in-out;
+}
+
+.CardContent-top {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.kebab {
+  width: 2rem;
+}
+
+.CardContent-elapsed-time {
+  font-size: 1.3rem;
+  color: var(--text-content-gray);
+}
+
+.CardContent-description {
+  height: 4.9rem;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  word-break: break-word;
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
+  font-size: 1.6rem;
+  line-height: 150%;
+}
+
+.CardContent-created-at {
+  font-size: 1.4rem;
+  color: var(--text-content-black);
+}
+
+.CardContent.CardContent-hovered {
+  background-color: var(--light-blue);
+}
```

isHovered props가 단순 UI를 위한 목적이라면 이 부분은 css에서 처리하는게 더 간단할 것 같아요!
```suggestion
.CardContent:hover {
  background-color: var(--light-blue);
}
```